src/box/lua/net_box.c

```diff
@@ -1294,10 +1294,26 @@ netbox_encode_commit_or_rollback(lua_State *L, enum iproto_type type, int idx,
 				 struct mpstream *stream, uint64_t sync,
 				 uint64_t stream_id)
 {
-	(void)L;
-	(void) idx;
 	assert(type == IPROTO_COMMIT || type == IPROTO_ROLLBACK);
 	size_t svp = netbox_begin_encode(stream, sync, type, stream_id);
+	if (type == IPROTO_COMMIT) {
+		bool has_is_sync = !lua_isnone(L, idx);
```

Comment:
`lua_isnoneornil` should work just fine.

Reply:
Done.